mbuild/formats/gomc_conf_writer.py

```diff
@@ -1878,43 +1939,78 @@ class GOMCControl:
+        file (checkpoint.dat) or not. Restarting the simulation with checkpoint.dat
+        would result in an identical outcome, as if previous simulation was continued.
+    Parameters : str, (default=None)
+        Override all other force field directory and filename input with the correct extension (.inp or .par).
+        Note: the default directory is the current directory with the Charmm object file name.
+    Coordinates_box_0 : str, (default=None)
+        Override all other box 0 pdb directory and filename inputs with the correct extension.
+        Note: the default directory is the current directory with the Charmm object file name.
+    Structure_box_0 : str, (default=None)
+        Override all other box 0 psf directory and filename inputs with the correct extension.
+        Note: the default directory is the current directory with the Charmm object file name.
+    Coordinates_box_1 : str, (default=None)
+        Override all other box 1 pdb directory and filename inputs with the correct extension.
+        Note: the default directory is the current directory with the Charmm object file name.
+    Structure_box_1 : str, (default=None)
+        Override all other box 1  psf directory and filename inputs with the correct extension.
+        Note: the default directory is the current directory with the Charmm object file name.
+    binCoordinates_box_0 : str, (default=None)
+        The box 0 binary coordinate file is used only for restarting a GOMC simulation,
+        which provides increased numerical accuracy.
+    extendedSystem_box_0 : str, (default=None)
+        The box 0 vectors and origin file is used only for restarting a GOMC simulation.
+    binVelocities_box_0 : str, (default=None)
+        The box 0 binary velocity file is used only for restarting a GOMC simulation,
+        which provides increased numerical accuracy. These velocities are only passed thru
+        GOMC since Monte Carlo simulations do not utilize any velocity information.
+    binCoordinates_box_1 : str, (default=None)
+        The box 1 binary coordinate file is used only for restarting a GOMC simulation,
+        which provides increased numerical accuracy.
+    extendedSystem_box_1 : str, (default=None)
+        The box 1 vectors and origin file is used only for restarting a GOMC simulation.
+    binVelocities_box_1 : str, (default=None)
+        The box 1 binary velocity file is used only for restarting a GOMC simulation,
+        which provides increased numerical accuracy. These velocities are only passed thru
+        GOMC since Monte Carlo simulations do not utilize any velocity information.
     input_variables_dict: dict, default = None
         These input variables are optional and override the default settings.
         Changing these variables likely required for more advanced systems.
         The details of the acceptable input variables for the selected
         ensembles can be found by running this python workbook,
```

Comment:
I think this was leftover from a notebook. Sounds like there is supposed to be a notebook here?

Reply:
I removed the notebook part was removed as it may not be general.  I meant run the command listed in python and it will yield the options for the user, if they don't know them.  The variable was slightly modified as below:

"input_variables_dict: dict, default = None
        These input variables are optional and override the default settings.
        Changing these variables likely required for more advanced systems.
        The details of the acceptable input variables for the selected
        ensembles can be found by running the code below in python,
        >>> print_valid_ensemble_input_variables('GCMC', description = True)
        which prints the input_variables with their subsection description
        for the selected 'GCMC' ensemble (other ensembles can be set as well)."